src/psbt.js

```diff
@@ -163,6 +174,17 @@ class Psbt {
     return this;
   }
   addOutput(outputData) {
+    if (
+      arguments.length > 1 ||
+      !outputData ||
+      outputData.value === undefined ||
```

Comment:
Just a personal preference but I'd always opt for `typeof foo === 'undefined'` over `foo === undefined`.

Two reasons off the top of my head:

1. If variable is not defined typeof will still work, strict comparison will throw. (not an issue here with object properties but nice to be consistent)
2. `undefined` value can be redefined as something else causing this check to fail.

Both pretty pedantic reasons that clearly won't affect this code so totally understand if you wanna stick with a strict equality check.

Just my 2c as to why I'd always avoid `=== undefined`.

Reply:
That makes sense.

Reply:
Maybe it's good to consider migrating the way of checking undefined sometime.